src/xunit.performance.api/XunitBenchmark.cs

```diff
@@ -14,23 +14,25 @@ static XunitBenchmark()
             // TODO: Read this from file.
             //   1. This reads as: flags read from config file based on running platform!
             //   2. Format could be something like this: `{ "Name" : "BranchMispredictions", "interval" : "100000" }`
-            RequiredProviders = new ProviderInfo[]
+            PerformanceMonitorCounters = new List<IPerformanceMonitorCounter>
+            {
+                new BranchMispredictionsPerformanceMonitorCounter(),
+                new CacheMissesPerformanceMonitorCounter(),
+            };
+
+            PerformanceMetricInfos = new List<BasePerformanceMetric>
```

Comment:
OK, particularly if this is to go away in the future, and since the lists only have two elements each, this looks reasonable.